deployment/ccip/changeset/deployer_group.go

```diff
@@ -18,11 +18,27 @@ import (
 
 	"github.com/smartcontractkit/chainlink/deployment"
 	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+	deploymenttypes "github.com/smartcontractkit/chainlink/deployment/common/types"
 )
 
 // MCMSConfig defines timelock duration.
 type MCMSConfig struct {
 	MinDelay time.Duration
+	MCMSType deployment.ContractType
```

Comment:
@kylesmartin I believe it should be sufficient to detect emergency or cancellation flow. Am I missing something more?